googlemaps/client.py

```diff
@@ -71,12 +79,12 @@ def __init__(self, key=None, client_id=None, client_secret=None,
             seconds.
         :type retry_timeout: int
 
-        :param client_id: (for Maps API for Work customers) Your client ID.
-        :type client_id: string
-
-        :param client_secret: (for Maps API for Work customers) Your client
-            secret (base64 encoded).
-        :type client_secret: string
+        :param queries_per_second: Manually specify the number of queries
```

Comment:
Cool - should we do a major version bump for this then considering the behaviour change?